src/data-loader.ts

```
@@ -656,9 +680,9 @@
 			const modelsUsed = extractUniqueModels(entries, e => e.model);
 
 			return {
-				date,
+				date: createDailyDate(date),
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)